src/planscape/forsys/get_forsys_inputs.py

```diff
@@ -20,26 +23,37 @@ class ForsysInputHeaders():
     # Only used for generation, not prioritization.
     FORSYS_GEO_WKT_HEADER = "geo"
 
+    # Note: when each stand represents a pixel, conditions and priorities will
+    # have the same values; however, when a single stand represents multiple
+    # pixels, conditions and priorities will have diffent values.
+    # Conditions represent the mean score across all pixels within a stand.
+    # This is useful when coloring maps.
```

Comment:
when you say useful, do you mean that is how it is used? can we be specific about that

Reply:
Done.